### ISSUE TITLE
Libsecret and zypak needed ?

### ISSUE THREAD

**Issue by larina3315** [flathub/com.vscodium.codium#315]
https://github.com/flathub/com.visualstudio.code/pull/498
https://github.com/flathub/com.visualstudio.code/pull/497

Referencing these two issues from VSC, I was wondering if VSCodium also used similar techniques and should they be considered for removal/modification.

**Comment by noonsleeper** [comment]
@larina3315  thanks, I will try using 
```
base: org.electronjs.Electron2.BaseApp
base-version: '23.08'
```
if this work, then we can remove those, like you mention

**Comment by larina3315** [comment]
Whether this fixes the sync issue or not also needs to be tested.
If the sync issue is fixed, then I'll close https://github.com/flathub/com.vscodium.codium/pull/294

**Comment by noonsleeper** [comment]
This was addressed by #319
